binary/pom.xml

```diff
@@ -117,7 +117,7 @@
                         <configuration>
                             <tasks>
                                 <echo message="Downloading JRE..."/>
-                                <get src="https://www.dropbox.com/s/68r915h7hxaq5ms/jre.tar.gz?dl=1"
+                                <get src="https://www.dropbox.com/scl/fi/2rjaso3cxgab0jdjxwpjr/jre.tar.gz?dl=1"
```

Comment:
@david7378 the check is still failing. Could the reason be that the asset doesn't have the right permissions on Drive?

I'm asking because if you try to open the URL directly the authentication is requested.

Reply:
I checked the permissions, and they are set to allow anyone with the link to access it. However, the only way I could access the link is with the key in the url. 
```
https://www.dropbox.com/scl/fi/2rjaso3cxgab0jdjxwpjr/jre.tar.gz?rlkey=1wrf6mcemn5d2e88t8u0wxbk3&dl=1
```
But Maven complains because the link contains the symbol "=" which causes an issue..
![image](https://github.com/geosolutions-it/MapStore2/assets/88064044/1e26026e-3247-45d8-800a-a9c8fe2c5e5f)

Reply:
Finally I used this url ```https://www.dropbox.com/scl/fi/2rjaso3cxgab0jdjxwpjr/jre.tar.gz?rlkey=1wrf6mcemn5d2e88t8u0wxbk3&amp;dl=1```, and all tests have passed.